praw/models/reddit/collections.py

```diff
@@ -441,14 +441,18 @@ def __init__(
         super().__init__(reddit, _data)
         self.subreddit_fullname = sub_fullname
 
-    def create(self, title: str, description: str):
+    def create(self, title: str, description: str, display_layout: str):
         """Create a new :class:`.Collection`.
 
         The authenticated account must have appropriate moderator permissions in the
         subreddit this collection belongs to.
 
         :param title: The title of the collection, up to 300 characters.
         :param description: The description, up to 500 characters.
+        :param display_layout: Either ``"TIMELINE"`` for events or discussions or
+            ``"GALLERY"`` for images or memes. Passing ``""`` or ``None`` will make the
+            collection appear on Reddit as if ``display_layout`` is set to
+            ``"TIMELINE"``.
```

Comment:
```suggestion
        :param display_layout: Either ``"TIMELINE"`` for events or discussions or
            ``"GALLERY"`` for images or memes. ``None`` will make the collection appear
            on Reddit as if this is set to ``"TIMELINE"`` (default:``None``).
```

Reply:
Why take out empty string?

Reply:
It doesn't make much sense to pass an empty string when `None` makes more sense to clear the set layout.